About.html

```diff
@@ -4,17 +4,147 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style/about.css">
+    <link rel="stylesheet" href="./dist/output.css">
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script src="about.js" defer></script>
+    <title>Cardano Student Seminar Home page</title>
 </head>
 <body>
     <header>
-        <nav></nav>
+        <nav>
+            <div class="desktop-toptag md:h-8 md:block hidden">
+                <ul class="flex text-white gap-5 justify-end pr-96 items-center pt-1">
+                    <li><a href=""><img class="h-5" src="images/facebook.png" alt="facebook icon"></a></li>
+                    <li><a href=""><img class="h-5" src="images/twitter.png" alt="facebook icon"></a></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="md:flex items-center pl-64 hidden bg-white drop-shadow-sm">
+            <a href="index.html"><img src="images/project.png" class="h-20" alt=""></a>
+             <ul class="flex gap-16 ml-72 text-base font-medium">
+                 <li><a href="">About</a></li>
+                 <li><a href="">Program</a></li>
+                 <li><a href="">Join</a></li>
+                 <li><a href="">Sponsor</a></li>
+                 <li><a href="">News</a></li>
+             </ul>
+             <button type="button" class="ml-28 border-4 border-brandorange text-brandorange p-3">CC Campaign</button>
+          </div>
+        </nav>
```

Comment:
- [ ] Kindly your design should be responsive on the mobile and desktop versions, so the navbar makes an issue with some widths, Kindly fix it to make your design perfect. (in about & index pages)

![image](https://user-images.githubusercontent.com/35535130/168820593-b2544c19-4eea-4054-9d04-87ff5e7fc317.png)